components/earnings/commons/earnings_base.lua

```diff
@@ -41,16 +42,22 @@ function Earnings.calculateForPlayer(args)
 		player = mw.ext.TeamLiquidIntegration.resolve_redirect(player)
 	end
 
+	-- since TeamCards on some wikis store players with underscores and some with spaces
+	-- we need to check for both options
+	playerAsPageName = player:gsub(' ', '_')
```

Comment:
```suggestion
	local playerAsPageName = player:gsub(' ', '_'):gsub('_', ' ')
```

and remove lines 36 and 48

Reply:
why gsub ` ` to `_` and then back again?

basically i need for an entry of `AA_BB CC` to get
`player = AA BB CC`
`playerAsPageName = AA_BB_CC`

with the suggestion i would get
`player = AA BB CC`
`playerAsPageName = AA BB CC`

i will move the first `gsub` (for player) into an `else` of the resolve redirect since redirect resolve already returns it with spaces instead of underscores

Reply:
> why gsub ` ` to `_` and then back again?

That was your code, I just simplified it. 😉